packages/dnb-eufemia/src/components/upload/UploadFileListCell.tsx

```diff
@@ -26,7 +26,7 @@ import { UploadFile, UploadFileNative } from './types'
 
 // Shared
 import { getPreviousSibling } from '../../shared/component-helper'
-import useUpload from './useUpload'
+import useUpload, { isFilesDuplicates } from './useUpload'
```

Comment:
```suggestion
import useUpload, { hasFilesDuplicates } from './useUpload'
```

I'm not sure. But "has" sounds more correct in that case.

Reply:
Agree, will rename tomorrow :)

Reply:
I tried improving it by renaming to `isFileEqual`, hope that makes it better 🤔